build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -252,54 +252,43 @@ public static BuildTargetResult CompileCoreHost(BuildTargetContext c)
             return c.Success();
         }
 
-        [Target(nameof(CompileTargets.GenerateStubHostPackages))]
-        public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
+        [Target]
+        public static BuildTargetResult GenerateMSbuildPropsFile(BuildTargetContext c)
         {
             var arch = IsWinx86 ? "x86" : "x64";
-            
+            var hostVersion = c.BuildContext.Get<HostVersion>("HostVersion");
+
+            var msbuildProps = new StringBuilder();
+
+            msbuildProps.AppendLine(@"<Project xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">");
+            msbuildProps.AppendLine("  <PropertyGroup>");
+            msbuildProps.AppendLine($"    <Platform>{arch}</Platform>");
+            msbuildProps.AppendLine($"    <DotNetHostBinDir>{Dirs.CorehostLatest}</DotNetHostBinDir>");
+            msbuildProps.AppendLine($"    <HostVersion>{hostVersion.LatestHostPolicyVersion.WithoutSuffix}</HostVersion>");
+            msbuildProps.AppendLine($"    <HostResolverVersion>{hostVersion.LatestHostFxrVersion.WithoutSuffix}</HostResolverVersion>");
+            msbuildProps.AppendLine($"    <HostPolicyVersion>{hostVersion.LatestHostVersion.WithoutSuffix}</HostPolicyVersion>");
+            msbuildProps.AppendLine($"    <BuildNumberMajor>{hostVersion.LatestHostBuildMajor}</BuildNumberMajor>");
+            msbuildProps.AppendLine($"    <BuildNumberMinor>{hostVersion.LatestHostBuildMinor}</BuildNumberMinor>");
```

Comment:
> We can update the Microsoft.NETCore.App.pkgproj to set that and exclude the "-00". I will preemptively create a PR with that change if that helps others in consuming this.

We are down the path of `Microsoft.NETCore.App` containing the "-00" suffix already. Let's stick with what we have.